source/Octopus.Tentacle.Tests/Integration/ScriptLogFixture.cs

```diff
@@ -65,6 +65,27 @@ public void ShouldAppend()
                 Assert.That(logs[0].Source, Is.EqualTo(ProcessOutputSource.StdErr));
             }
         }
+        
+        [Test]
+        public void ShouldHandleIncompleteLine()
+        {
+            using (var appender = sut.CreateWriter())
+            {
+                appender.WriteOutput(ProcessOutputSource.StdOut, "Hello");
+                appender.WriteOutput(ProcessOutputSource.StdOut, "World");
+            }
+
+            using (var fs = File.Open(logFile, FileMode.Open))
```

Comment:
2 nits:
- I still like using the full name. I.e. fs => fileStream
- No `var` for `FileInfo`